newm-server/src/test/kotlin/io/newm/server/grpc/GrpcTests.kt

```diff
@@ -16,7 +21,17 @@ class GrpcTests {
     fun `test queryUtxos`() = runBlocking {
         // plainText for localhost testing only. use SSL later.
         val channel = ManagedChannelBuilder.forAddress("localhost", 3737).usePlaintext().build()
-        val client = NewmChainGrpcKt.NewmChainCoroutineStub(channel)
+        val client =
+            NewmChainGrpcKt.NewmChainCoroutineStub(channel).withInterceptors(
+                MetadataUtils.newAttachHeadersInterceptor(
+                    Metadata().apply {
+                        put(
+                            Metadata.Key.of("Authorization", Metadata.ASCII_STRING_MARSHALLER),
+                            "Bearer <JWT_TOKEN_HERE_DO_NOT_COMMIT>"
```

Comment:
Was this a reminder to yourself to not commit like that?

Reply:
These tests are designed to run manually. You have to put in a real JWT token to run the test. This is a reminder to anybody in the future who wants to run the test to not commit it when they put a real JWT in there.